extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -378,6 +378,13 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 			containers.push(this.createLabelInputContainer(localizedConstants.ContainmentTypeText, containmentDropbox));
 		}
 
+		if (!isUndefinedOrNull(this.objectInfo.isLedgerDatabase)) {
+			let isLedgerDatabaseInput = this.createCheckbox(localizedConstants.IsLedgerDatabaseText, async () => {
```

Comment:
Remove this async? I don't see any awaits in the block

Reply:
The callback argument to createCheckbox (and many others) always expects a Promise return type